build.gradle

```diff
@@ -1,5 +1,6 @@
 plugins {
     id 'java'
+    id "org.owasp.dependencycheck" version "11.1.0"
```

Comment:
Please, move version to (https://github.com/DimaGalchenko/winter-framework/blob/develop/version.gradle) file

Reply:
I am not sure that will work. `applyFrom: version.gradle` goes after plugins block

Reply:
According to PR(adding Jacoco) it should works:
<img width="256" alt="Screenshot 2024-11-04 at 09 50 36" src="https://github.com/user-attachments/assets/ff238ba2-24c2-43be-9e91-64ceb3b27651">
But I am not sure

Reply:
Added OWASP Dependency Check plugin to `build.gradle` using an externalized version reference in `version.gradle`. Updated the `plugins` block to dynamically load the version from the `versions` map in `version.gradle`.